### PR TITLE
docs: fix typo

### PR DESCRIPTION
**Pull request by jjangga0214**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[x] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

### CODE REVIEW

**Comment by kecrily** [comment]
Ah, I found this when I was translating it. But I couldn't quite read it, and I wasn't sure if it was a typo or my English wasn't good enough. Then I got busy and forgot about it. Waiting for an answer 🤔

File: docs/src/user-guide/configuring/configuration-files-new.md [hunk 1]
```diff
@@ -561,5 +561,3 @@ npx eslint -c some-other-file.js **/*.js
 ```
 
 In this case, ESLint will not search for `eslint.config.js` and will instead use `some-other-file.js`.
-
-Each configuration file exports one or more configuration object. A configuration object
```
Comment (snitin315) [inline docs/src/user-guide/configuring/configuration-files-new.md:565]: Not sure, maybe more lines were meant to be added here?  cc @nzakas

Comment (nzakas) [reply docs/src/user-guide/configuring/configuration-files-new.md:565]: It looks like this was copy-pasted from earlier in the page, so it’s safe to remove.

**Review by nzakas — APPROVED** [review]

**Review by snitin315 — APPROVED** [review]

**Comment by snitin315** [comment]
Bumping CLA

**Review by mdjermanovic — APPROVED** [review]
LGTM, thanks!

**Pull request merged**